acesso-io/acessobio-ios
# add frame cpf

> Why?
Equalize iOS with js

> Changes
add document frame CPF

> How to test
select frame documents type CPF and take picture

> Preview
<img src="https://user-images.githubusercontent.com/78861681/136447948-ceebe308-c528-45b9-b595-187afe31bdf6.png" height="500" />


Issues
https://github.com/acesso-io/squad-check-mobile/issues/223
